src/EquationSystems.C

```diff
@@ -46,6 +50,71 @@
 namespace sierra{
 namespace nalu{
 
+namespace {
+
+inline void init_connectivity_helper(Realm& realm, Teuchos::RCP<CrsGraph>& graph)
+{
+  // This graph is not active, return early
+  if (graph == Teuchos::null) return;
+
+  graph->buildNodeGraph(realm.interiorPartVec_);
+  graph->buildElemToNodeGraph(realm.interiorPartVec_);
+  graph->buildFaceElemToNodeGraph(realm.bcPartVec_);
+
+  if (realm.hasNonConformal_)
+    graph->buildNonConformalNodeGraph(stk::mesh::PartVector());
+  if (realm.hasOverset_)
+    graph->buildOversetNodeGraph(stk::mesh::PartVector());
+}
+
+template<typename LambdaFunc>
+void initialize_connectivity(
+  Realm& realm,
+  std::vector<EquationSystem*>& eqSysVec,
+  LambdaFunc func)
+{
+  std::map<int, std::vector<EquationSystem*>> eqSysMap;
+  // Group equation systems based on numDof. Note that this is safe for
+  // reinitialization also as the `linsys_` instance is still valid and does not
+  // get reset until the call to `EquationSystem::reinitialize_linear_system`
+  for (auto* eqsys: eqSysVec) {
+    // Skip wrapper type equation systems that don't have a LinearSystem instance
+    if (eqsys->linsys_ == nullptr) continue;
```

Comment:
@jhux2 This is where the regression was introduced. We should change this line to 

```C++

// For "wrapper" equation systems, execute their default initialize() method to allow convergence tolerance 
// to be passed to their children
if (eqsys->linsys_ == nullptr)
  eqsys->initialize();
```

This will allow you to remove the stuff had to add in `momentum` and `continuity`

Reply:
This change will allow us to also handle `SST` wrapper 

https://github.com/Exawind/nalu-wind/blob/b9d2834c770be4786e23f1d32e28f718af82c991/src/ShearStressTransportEquationSystem.C#L110-L116

Reply:
@sayerhs Thanks, that fixed it!  We're just left with just a few failures, which seem benign.

Is this ok to merge?

```
6: FAIL: airfoilRANSEdgeTrilinos.................     4.7904s 1.4686e-01 2.2813e-07
22: FAIL: ductWedge...............................     0.6451s 5.2951e-11 4.2101e-07
23: FAIL: edgeHybridFluids........................    35.4491s 3.1472e-14 3.5561e-11
24: FAIL: edgePipeCHT.............................     5.6124s 1.8287e-12 2.3133e-12
25: FAIL: ekmanSpiral.............................    12.8755s 9.8148e-07 8.9743e-05
43: FAIL: karmanVortex............................     0.5497s 2.5751e-08 9.4357e-07
50: FAIL: nonIsoEdgeOpenJet.......................    15.8953s 5.6479e-11 1.5776e-07
53: FAIL: nonIsoNonUniformEdgeOpenJet.............    43.9456s 4.0505e-08 2.2234e-06
```